portal-bridge/src/bridge/history.rs

```diff
@@ -341,25 +355,12 @@ impl HistoryBridge {
         &self,
```

Comment:
I'd also get rid of this function and just inline the relevant code since there's no more gossiping going on